core/msg.c

```diff
@@ -490,6 +492,8 @@ void msg_init_queue(msg_t *array, int num)
     cib_init(&(me->msg_queue), num);
 }
 
+#define MSG_QUEUE_PRINT_MAX 16U
```

Comment:
```suggestion
#ifndef CONFIG_MSG_QUEUE_PRINT_MAX
#define CONFIG_MSG_QUEUE_PRINT_MAX 16U
#endif
```

makes it a bit easier to change this

Reply:
I know :see_no_evil: done now